site/content/integrate/apps/structure/manifest/_index.md

```diff
@@ -57,40 +64,40 @@ An example manifest looks like this:
 
 ### Permissions
 
-Apps need to request permissions from the admin who is installing them. After an app is installed and available to end users, each user in Mattermost will need to "authenticate" with your app. Using the `act_as_user` permission, your app will be performing an API call on each authenticated user's behalf.
-
-Below are the permissions ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Permission)) an app can currently ask for:
+Apps need to request permissions from the System Admin who is installing them. After an App is installed and available to end users, each user in Mattermost will need to "authenticate" with your App.
```

Comment:
Heads up: https://github.com/mattermost/mattermost-plugin-apps/pull/367 will introduce OAuth2 API scopes, and remove unused permissions.

In general, I'm 2/5 bent on eventually eliminating Permissions in favor of Scopes.

Reply:
Thanks for the heads-up :slightly_smiling_face:
We will include a revisit of this topic when documenting the next release of the framework.